### PR TITLE
[Merged by Bors] - TY-2919 Change order of ranking the documents, fix ranking of active search & trending topics

### PR DESCRIPTION
**Pull request by Robert-Steiner**
Ticket: 
- [TY-2919]

- adjusted the rank test (with the previous embeddings, all scores were the same)
- I don't expect any major performance losses by reversing the vec after each stack rank but an improvement could be to add an additional parameter to the rank method that specifies if the function should rank in descending or ascending order.

[TY-2919]: https://xainag.atlassian.net/browse/TY-2919?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by finiteprods — COMMENTED** [review]
>an improvement could be to add an additional parameter to the rank method that specifies if the function should rank in descending or ascending order

I agree this would be better, also to see more clearly which parts need which order. but i don't think it needs to be done here.

File: discovery_engine_core/ai/ai/src/ranker/system.rs [hunk 1]
```diff
@@ -95,7 +95,7 @@ impl Ranker {
         self.kpe.run(sequence).map_err(Into::into)
     }
 
-    /// Ranks the given documents based on the learned user interests.
+    /// Ranks the given documents in descending order based on the learned user interests.
```
[the remark this answers is not in the document]
Comment (Robert-Steiner) [reply discovery_engine_core/ai/ai/src/ranker/system.rs]: done

**Review by finiteprods — APPROVED** [review]
:+1:

**Comment by Robert-Steiner** [comment]
bors merge

**Pull request closed without merging**